src/offset/utc.rs

```diff
@@ -3,7 +3,14 @@
 
 //! The UTC (Coordinated Universal Time) time zone.
 
+use super::{FixedOffset, LocalResult, Offset, TimeZone};
```

Comment:
Please don't make unrelated changes (the imports are sorted and split this way on purpose).

Reply:
apologies, overbearing IDE.